views/home.erb

```diff
@@ -3,19 +3,27 @@
   <head>
     <meta charset='UTF-8' />
     <title>Hangman</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Comfortaa|Inconsolata" rel="stylesheet">
   </head>
-  <body class='container'>
-    <h1> WELCOME TO HANGMAN GAME</h1>
-    <img src='<%= @image %>.jpg' alt='hangman image' height="500" width="500">
-    <p><%= @secret_word %></p>
-    <form action='/play' method='post'>
-      <label for='letter'>Make a guess:</label>
-      <input type='text' id='letter' name='letter_chosen' autofocus="autofocus">
-      <button type='submit' class='btn btn-primary'>submit</button>
-    </form>
-    <p><%= @letters %></p>
-    <p><%= @show_word %></p>
-    <a href='/games-won'>How many times did I win?</a>
+  <body>
+    <div class='container'>
+      <div class='link'>
+	<a href='/games-won'>Win counter</a>
+      </div>
+      <h1><b> Guess the word! </b></h1>
```

Comment:
you shouldn't use the `b` tag to make something look bold. Here is some explanation. Look at the third bullet point in particular. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b